src/main/resources/bank/items/.gitignore

```diff
@@ -0,0 +1,3 @@
+# ignore everything except .gitignore
```

Comment:
I dislike defining .gitignores at different places throughout a project.  What value does having the directory provide?